Week3/week3HomeWork/Exercise1/amazingNaji.js

```diff
@@ -0,0 +1,22 @@
+function giveCompliment(myName) {
+    const compliment = [
```

Comment:
I would change the name from `compliment` to `compliments` (because there are multiple items in the `array`)